guides/source/active_record_callbacks.md

```diff
@@ -621,6 +623,15 @@ end
 
 NOTE: This applies to all `after_*_commit` variations too, such as `after_destroy_commit`.
 
+This order can be set via configuration:
+
+```ruby
+config.active_record.run_after_transaction_callbacks_in_order_defined = false
+```
+
+If `true` (the default from Rails 7.1), callbacks are executed in the order they
+are defined. If `false`, the order is reversed, just like in the example above.
```

Comment:
```suggestion
When set to `true` (the default from Rails 7.1), callbacks are executed in the order they
are defined. When set to `false`, the order is reversed, just like in the example above.
```

Reply:
Thanks! That reads better indeed.
I'll also change it in the API docs where I copied it from.